test/test_torch.py

```diff
@@ -1707,6 +1707,11 @@ def test_add(self):
             expected = torch.zeros(2, 3, device=device).bool()
             self.assertEqual(res, expected)
 
+            # bfloat16
+            m1 = torch.tensor([1, 2], dtype=torch.bfloat16)
```

Comment:
nit: wouldn't `torch.tensor([1., 2.], dtype=torch.bfloat16)` be the canonical representation?